packages/documentation-site/patternfly-docs/content/design-guidelines/components/table/table.md

```diff
@@ -118,6 +118,32 @@ The actionable table provides checkboxes or radio buttons that enable users to s
 #### When not to use
 * Users can not take any actions on table items/rows.
 
+### Comparison table
```

Comment:
should we also add this as a list item on line 77?

Reply:
great catch!! Just updated